projects/components/src/lib/date-time/date-time.component.ts

```diff
@@ -5,83 +5,97 @@ import { TimeValue } from './time-value';
 import { MatDatepicker } from '@angular/material/datepicker';
 
 @Component({
-  selector: 'mz-datetime',
-  templateUrl: './date-time.component.html',
-  styleUrls: ['./date-time.component.css'],
-  providers: [{
-    provide: NG_VALUE_ACCESSOR,
-    useExisting: forwardRef(() => DateTimeComponent),
-    multi: true
-  }]
+    selector: 'mz-datetime',
+    templateUrl: './date-time.component.html',
+    styleUrls: ['./date-time.component.css'],
+    providers: [
+        {
+            provide: NG_VALUE_ACCESSOR,
+            useExisting: forwardRef(() => DateTimeComponent),
+            multi: true,
+        },
+    ],
 })
 export class DateTimeComponent implements ControlValueAccessor {
+    date: Date | undefined;
+    time: String | undefined;
+    private tempDate: string | undefined | null = null;
+    private tempTime: string | null = null;
+    type = 'outline';
+    @ViewChild('datePicker') datepicker!: MatDatepicker<Date>;
 
-  date: Date | undefined;
-  time: String | undefined;
-  private tempDate: string | undefined | null = null;
-  private tempTime: string | null = null;
-  type = 'outline';
-  @ViewChild('datePicker') datepicker!: MatDatepicker<Date>;
+    @Input()
+    min?: Date | undefined;
 
-  @Input()
-  min?: Date | undefined;
+    @Input()
+    max?: Date | undefined;
 
-  @Input()
-  max?: Date | undefined;
+    @Input()
+    disabled?: boolean;
```

Comment:
Unless `null` behaves differently from `false`, I think we can just default this to `false` to avoid dealing with 3 values.